lib/fs.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+async function mkdirp(pathName, callback) {
+  const version = process.versions.node.split('.').map(el => parseInt(el));
+  if (version[0] < 10 || (version[0] === 10 && version[1] <= 11)) {
```

Comment:
It will be better to move `mkdirp` polyfill to another function since now it will check the version on every iteration.